Dockerfile

```diff
@@ -1,6 +1,6 @@
 # syntax=docker/dockerfile:1
 
-FROM python:3.10
+FROM python:3.10-slim
```

Comment:
> This image does not contain the common packages contained in the default tag and only contains the minimal packages needed to run python. Unless you are working in an environment where only the python image will be deployed and you have space constraints, we highly recommend using the default image of this repository.

Are you sure that we're able to use `slim` fine?

Reply:
Seemed to work in my brief local testing. We're not relying on much OS stuff so it should be fine.